### PR TITLE
Introduce toggle-able-agenda. Closes #21

### LINKED ISSUE #21: Reuse Aganda toggle from simplabs.com
introduced via simplabs/simplabs.github.io#57

### PR DESCRIPTION
**Pull request by ganeshbsub**

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
Looks good - some comments:

* The style of the button looks a bit out of place - I guess the purple color might work better or we could style it like the newsletter submission form button or like the _"Register now"_ link.
* There needs to be some spacing between the _"Agenda"_ buttons and the _"Day 2"_ and _"Day 3"_ headings
* The pointer must always be `pointer` over the buttons

**Comment by ganeshbsub** [comment]
@marcoow how about this current button?

File: css/main.scss [hunk 1]
```diff
@@ -235,6 +235,10 @@ a.anchor {
     text-decoration: underline;
   }
 
+  .day {
+    margin-top: 20px;
```
Comment (marcoow) [inline css/main.scss]: Shall we use the same 80px that the content is offset from the surrounding container?

**Review by marcoow — COMMENTED** [review]
The button is nice, I think we can improve the spacing a bit though.

File: css/main.scss (hunk 1 above)
Comment (ganeshbsub) [reply css/main.scss]: That works. Looks more uniform now, I guess.

**Review by marcoow — APPROVED** [review]

**Pull request merged**